src/main/app/controller/GetController.ts

```diff
@@ -124,6 +192,59 @@ export class GetController {
     });
   }
 
+  /**Cookies prefrences saver */
+
+  public CookiePrefrencesChanger = (req: AppRequest, res: Response): void => {
+    //?analytics=off&apm=off
+    if (req.query.hasOwnProperty('analytics') && req.query.hasOwnProperty('apm')) {
+      let cookieExpiryDuration = Number(config.get('cookies.expiryTime'));
+      const TimeInADay = 24 * 60 * 60 * 1000;
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      cookieExpiryDuration = cookieExpiryDuration * TimeInADay; //cookie time in milliseconds
+      const CookiePreferences = {
+        analytics: '',
+        apm: '',
+      };
+      if (req.query.hasOwnProperty('analytics')) {
+        switch (req.query['analytics']) {
+          case 'off':
+            CookiePreferences['analytics'] = 'off';
```

Comment:
for 'off' and 'on' creating constant and using it will be good for maintainability perspective.

Reply:
has been resolved